packages/angular-test/src/boolean.ts

```diff
@@ -70,7 +70,6 @@ export const booleanBaseTest = <C extends JsonFormsControl, I>
         component.ngOnInit();
         expect(component.data).toBe(true);
         expect(checkboxInstance.checked).toBe(true);
-        expect(checkboxInstance.disabled).toBe(false);
```

Comment:
why did you remove this, the idea was to check the default behavior is correct

Reply:
Restored as well..